backend/api/v1/models.py

```diff
@@ -1,21 +1,28 @@
+from sqlalchemy.sql import expression
+
 from app import db
 
+
 IdentifierType = db.String(32)
 
-ContentInstructorAssociations = db.Table("content_instructor_association", db.metadata,
-    db.Column('content_slug', IdentifierType, db.ForeignKey('contents.slug')),
-    db.Column('instructor_slug', IdentifierType, db.ForeignKey('instructors.slug'))
+ContentInstructorAssociations = db.Table(
+    "content_instructor_association",
+    db.metadata,
+    db.Column("content_slug", IdentifierType, db.ForeignKey("contents.slug")),
```

Comment:
Shouldn't these columns be non-nullable?

Reply:
Jep